.github/workflows/tpu-tests.yml

```diff
@@ -111,21 +111,15 @@ jobs:
 
     - name: Create node
       id: tpu-create
+      # TPU capacity is very limited so this workflow's success is optional. continue normally if creation fails
       continue-on-error: true
       env:
         JOB_NAME: ${{ env.PR_NUMBER }}-${{ matrix.pkg-name }}-${{ matrix.runtime }}-${{ env.SHA }}
       run: |
-        gcloud compute tpus tpu-vm create "$JOB_NAME" --accelerator-type=v4-8 --version="tpu-vm-v4-pt-$XLA_VER"
-
-    - name: Cancel job
-      if: steps.tpu-create.outcome != 'success'
-      env:
-        GH_TOKEN: ${{ github.token }}
-      run: |
-        gh run cancel ${{ github.run_id }}
```

Comment:
We cannot use this because it will cancel all jobs in the workflow. Not just the one that failed

Reply:
Then you can open each, it is just 4 option, so please keep it